.github/workflows/build.yml

```diff
@@ -1,24 +1,43 @@
-name: build
-on: push
+name: Build
+on:
+  push:
+    branches:
+      - main
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-    container:
-      image: openjdk:8-jdk
+    strategy:
+      matrix:
+        java: [
+          1.8, # Minimum
+          11,  # LTS
+          16,  # Latest
+        ]
+        os: [ubuntu-20.04]
+    runs-on: ${{ matrix.os }}
     steps:
-      - name: checkout repository
+      - name: Checkout repository
         uses: actions/checkout@v2
-      - name: validate gradle wrapper
+        with:
+          fetch-depth: 0
+      - name: Validate Gradle wrapper
         uses: gradle/wrapper-validation-action@v1
-      - name: build
+      - name: Setup JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Make Gradle wrapper executable
+        if: ${{ runner.os != 'Windows' }}
+        run: chmod +x ./gradlew
+      - name: Build
         uses: eskatos/gradle-command-action@v1
         with:
           arguments: checkLicenses build
           wrapper-cache-enabled: true
           dependencies-cache-enabled: true
-      - name: capture build artifacts
+      - name: Capture build artifacts
+        if: ${{ runner.os == 'Linux' && matrix.java == '11' }}
```

Comment:
Maybe capture artifacts for 8 - there *technically* is a backwards compatibility issue with `FloatBuffer#flip` (signature changed [but also somehow forwards compatible]), not that we're using that yet.

Reply:
resolved in c02a824